templates/v2/vault/relay/_list.html.twig

```diff
@@ -1,5 +1,8 @@
 <div class="container mx-auto w-75">
     {% for relay in relays %}
-        {{ include('v2/vault/relay/_card.html.twig', {relay}) }}
+        {{ app.user.beneficiaire
```

Comment:
Should not this be handled via permissions, with something like `is_granted('ROLE_PROFESSIONAL')` ?